modules/nested-tabs-html/widgets/nested-tabs-html.php

```diff
@@ -82,8 +82,10 @@ protected function register_controls() {
 		$tooltip_end = is_rtl() ? esc_html__( 'Left', 'elementor' ) : esc_html__( 'Right', 'elementor' );
```

Comment:
@rami-elementor Do you want to update these references for the user as well?

Reply:
Yes, I prefer no to use the term "Left"/"Right" anymore, not in the code and not in the Editor UI.

Reply:
@rami-elementor Do you have a suggestion of we can use instead? 'Start'/'End'?

Reply:
@hein-obox we can use `before` / `after`

Reply:
@rami-elementor I am not sure in what way, but I would prefer if you and Doron can come to a conclusion together.

Reply:
Implemented.